xled/control.py

```diff
@@ -432,7 +432,7 @@ def update_firmware(self, stage0, stage1):
                 fw_stage_sums[stage] = xled.security.sha1sum(stage1)
             log.debug("Firmware stage %d SHA1SUM: %r", stage, fw_stage_sums[stage])
             if not fw_stage_sums[stage]:
-                msg = "Failed to compute SHA1SUM for firmware stage %d.".format(stage)
+                msg = "Failed to compute SHA1SUM for firmware stage %d." % (stage)
```

Comment:
Why this change?

Reply:
I couldn't figure out why the previous line was giving the error and the second form wasn't. I'm happy to put it back if the linter passes it.

Reply:
Ah, maybe because `(stage)` is pretty much useless in this context and `stage` or `(stage,)` should have been used?

Reply:
I can monkey around with this and try to make it look the same if you want.